tests/test_redis_client.py

```diff
@@ -108,6 +109,17 @@ def test_should_call_set_if_enabled(mocked_redis_client):
     mocked_redis_client.redis_store.set.assert_called_with("key", "value", None, None, False, False)
 
 
+def test_should_not_call_get_cache_keys_by_pattern_if_not_enabled(mocked_redis_client):
+    mocked_redis_client.active = False
+    assert not mocked_redis_client.get_cache_keys_by_pattern("test-key-*")
+    mocked_redis_client.redis_store.keys.assert_not_called()
+
+
+def test_should_call_get_cache_keys_by_pattern_if_enabled(mocked_redis_client):
+    assert mocked_redis_client.get_cache_keys_by_pattern("test-key-*") == [b"test-1243", b"test-23423423"]
```

Comment:
If the parameter is a regular expression, I would not expect these keys to get returned as they would not fix `test-key-*`. A proper regex for the expected keys would be `test-[\d]+`. My impression is that the parameter format correspond more to a glob than a regex.

Reply:
you're probably right. It turns our I do not need this function anyways to make it work. So I removed it